api.md

```diff
@@ -58,6 +58,52 @@ your use case. Users will *rarely* want or need to use this command.
 converted to a hypertable (via `create_hypertable`), but must similarly
 be run only on an empty hypertable.
 
+**Space partitions**: The use of additional partitioning is a very
+specialized use case.  **Most users will not need to use it.**
+
+Space partitions use hashing: Every distinct item is hashed to one of
+*N* buckets.  Remember that we are already using (flexible) time
+intervals to manage chunk sizes; the main purpose of space
+partitioning is to enable parallel I/O to the same time interval.
+
+Parallel I/O can benefit in two scenarios: (a) two or more concurrent
+queries should be able to read from different disks in parallel, or
+(b) a single query should be able to use query parallelization to read
+from multiple disks in parallel.
+
+Note that query parallelization in PostgreSQL 9.6 (and 10) does not
```

Comment:
Maybe update this for PG11, which I believe has (at least some) support for parallel chunk/subtable scans.

Reply:
Perfect - Mike had the same comment. Updated!